en/SetupView/safety_ardupilot.md

```diff
@@ -0,0 +1,202 @@
+# Safety Setup (ArduPilot)
+
+The *Safety Setup* page allows you to configure failsafe settings for: low battery, GCS loss,  geofence, etc, and the failsafe actions.
+
+> **Tip** The setup page is slightly different for each of the vehicle types, reflecting the different safety settings in each platform.
+
+## Copter
+
+The Copter safety page is shown below. For more information see [Failsafe](ttp://ardupilot.org/copter/docs/failsafe-landing-page.html).
+
+![Safety Setup - Copter (Ardupilot)](../../assets/setup/safety/safety_arducopter.jpg)
+
+
+### Battery Failsafe {#battery_failsafe_copter}
+
+This panel sets the [Battery Failsafe](http://ardupilot.org/copter/docs/failsafe-battery.html) parameters.
+You can set low and critical thresholds for voltage and/or remaining capacity and define the action if the failsafe value is breached. 
+The thresholds can be disabled by setting them to zero.
+
+![Safety Setup - Battery1 Failsafe Triggers (Copter)](../../assets/setup/safety/safety_arducopter_battery1_failsafe_triggers.jpg)
+
+The configuration options are:
+- **Low action** ([BATT_FS_LOW_ACT](http://ardupilot.org/copter/docs/parameters.html#batt-fs-low-act-low-battery-failsafe-action)) - Select one of: None, Land, RTL, SmartRTL, SmartRTL or Land, Terminate.
+- **Critical action** ([BATT_FS_CRT_ACT](http://ardupilot.org/copter/docs/parameters.html#batt-fs-crt-act-critical-battery-failsafe-action)) - Select one of: None, Land, RTL, SmartRTL, SmartRTL or Land, Terminate.
+- **Low voltage threshold** ([BATT_LOW_VOLT](http://ardupilot.org/copter/docs/parameters.html#batt-low-volt-low-battery-voltage)) - Battery voltage that triggers the *low action*. 
+- **Critical voltage threshold** ([BATT_CRT_VOLT](http://ardupilot.org/copter/docs/parameters.html#batt-crt-volt-critical-battery-voltage))- Battery voltage that triggers the *critical action*. 
+- **Low mAh threshold** ([BATT_LOW_MAH](http://ardupilot.org/copter/docs/parameters.html#batt-low-mah-low-battery-capacity)) - Battery capacity that triggers the *low action*. 
+- **Critical mAh threshold** ([BATT_CRT_MAH](http://ardupilot.org/copter/docs/parameters.html#batt-crt-mah-battery-critical-capacity)) - Battery capacity that triggers the *critical action*. 
+
+
+<!-- Questions
+- What about Battery 2?
```

Comment:
There appear to be Battery 2 items in the parameters for copter. How are these supported? (ie if I have two batteries do I end up with two panels, or more sensibly, a combined failsafe?

Reply:
You end up with two panels. In order to get the second panel to show you need to enable the second batter on the Power page.

Reply:
Thanks, I'll tidy.

@rmackay9 - FYI, IMO doing safety settings individually on multiple batteries seems a little error prone- easy to end up getting low battery warnings when you have plenty of power in one battery but not the other.